src/services/notifications/payment.ts

```diff
@@ -23,13 +21,12 @@ export const transactionNotification = async ({
   logger,
   hash,
   txid,
+  usdPerSat,
 }: IPaymentNotification) => {
-  const satsPrice = await getCurrentPrice()
-
   let title
 
-  if (satsPrice) {
-    const usd = (amount * satsPrice).toFixed(2)
+  if (usdPerSat) {
+    const usd = (amount * usdPerSat).toFixed(2)
```

Comment:
toFixed is display logic - should probably be implemented client side or wrapped via a 'CurrencyFormatter' object - not critical as I can see in the diff this wasn't changed. Just an observation

Reply:
this value is shown in the notification, the mobile app cant update it